### ISSUE TITLE
Nothing happens after I've installed

### ISSUE THREAD

**Issue by ruben-cit** [sqlfluff/vscode-sqlfluff#82]
I've followed the installation instructions and nothing happens, either onSave or onEdit. Similar to what Anton and James raised in the [reviews](https://marketplace.visualstudio.com/items?itemName=dorzey.vscode-sqlfluff&ssr=false#review-details).

I ended up manually updating the `Preferences: User Settings` instead of the `Preferences: Settings` that is recommended (wasn't an option).

`sqlfluff` is installed and works from the command line.

I'm running:
- macOS Monterey v12.3 (Apple M1 chip)
- VSCode v1.74.2
- vscode-sqlfluff v2.2.2.
- sqlfluff v1.4.5

Please advise!

**Comment by RobertOstermann** [comment]
@ruben-cit Yeah I need to update the README to be `Preferences: User Settings`, I didn't realize that was changed in VSCode. Do you see anything in the SQLFluff output channel?

**Comment by mgreminger** [comment]
I was seeing the same symptoms. When using the suggested manual configuration in the README, the sqlfluff command fails if the `.sqlfluff` file doesn't exist. It started working for me when I removed the following line from the `settings.json`:

`"sqlfluff.config": "${workspaceFolder}/.sqlfluff",`

**Comment by ruben-cit** [comment]
@mgreminger Hmmm, interesting. On my end @RobertOstermann's suggestion to just look at the channel output resolved the issue for me! Guess I wasn't looking in the right place/the output wasn't being surfaced for w.e. reason.

**Comment by RobertOstermann** [comment]
@ruben-cit Does specifying an exact path work? You could do that in the workspace settings instead of the global settings. Not an ideal solution, just want to see if the problem is locating the workspace folder.

**Comment by AlgirdasJasinskas** [comment]
Also facing similar issue, just installed sqlfluff, but nothing happened. Preferences: User Settings does not even show up in the command palette when I try to search for it. MacOS.

**Comment by MinchoG** [comment]
> I was seeing the same symptoms. When using the suggested manual configuration in the README, the sqlfluff command fails if the `.sqlfluff` file doesn't exist. It started working for me when I removed the following line from the `settings.json`:
> 
> `"sqlfluff.config": "${workspaceFolder}/.sqlfluff",`

you are a lifesaver, thank you, I don't know why isn't this part of the docs - an empty .sqlfluff must be ignored and this checkbox removed
